packages/shared/src/utils/brand/brand.ts

```diff
@@ -0,0 +1,38 @@
+import config_data from '../../../brand.config.json';
+
+type TLandingCompany = {
+    fx: string;
+    malta: string;
+    maltainvest: string;
+    mx: string;
+    samoa: string;
+    svg: string;
+    v: string;
+};
+
+type TPlatform = {
+    name: string;
+    icon?: string;
+};
+
+type TPlatforms = {
+    trader: TPlatform;
+    dbot: TPlatform;
+    mt5: TPlatform;
+    dxtrade: TPlatform;
+    smarttrader: TPlatform;
+    bbot: TPlatform;
+    go: TPlatform;
+};
+
+export const getLegalEntityName = (landing_company: keyof TLandingCompany) => {
+    return config_data.legal_entities[landing_company];
+};
```

Comment:
just a clarification for these cases and further: do we need to set returned types for functions? I mean:
const func = ( {}: Tapams ): Treturnvalue => {
}

Reply:
@yauheni-kryzhyk-deriv i agree. it's better to declare a return type here so that the type of a return value is known in any places where we use it.

Reply:
Thank u guys for reviewing my PR.
we are using `getPlatformSettings` in `packages/shared/src/utils/platform/platform.ts`!
if we don't define return type for this we will get this error
```
Property 'icon' does not exist on type '{ name: string; icon: string; } | { name: string; icon: string; } | { name: string; icon: string; } | { name: string; icon: string; } | { name: string; icon: string; } | { name: string; icon: string; } | { name: string; }'.
  Property 'icon' does not exist on type '{ name: string; }'.ts(2339)
```
because of that I defined return type here!